llvm/test/tools/UpdateTestChecks/update_mc_test_checks/Inputs/amdgpu_asm.s.expected

```diff
@@ -0,0 +1,5 @@
+; NOTE: Assertions have been autogenerated by utils/update_mc_test_check.py UTC_ARGS: --version 5
+// RUN: llvm-mc -triple=amdgcn -show-encoding %s 2>&1 | FileCheck --check-prefixes=CHECK %s
+
+// CHECK: v_bfrev_b32_e32 v5, v1                  ; encoding: [0x01,0x71,0x0a,0x7e]
+v_bfrev_b32 v5, v1
```

Comment:
In asm tests we normally put check lines after instructions. Do you propose to change this? (I wouldn't mind either way.)

Reply:
It probably doesn't matter but I agree for ASM tests having it in the next line (or on the same line) makes more sense to me.

Reply:
I can put it to the next line. I think putting it on the same line might be a bit too long to read sometime

Reply:
done